platforms/enterprise/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/DevelocityBuildLifecycleServiceIntegrationTest.groovy

```diff
@@ -50,6 +52,41 @@ class DevelocityBuildLifecycleServiceIntegrationTest extends AbstractIntegration
         outputDoesNotContain("Configuring ':included")
     }
 
+    @IgnoreIf(
```

Comment:
We should use `Requires` with a precondition class, so we check that this test actually runs somewhere. In the mid-term we want to get rid of `@IgnoreIf`.

Reply:
Done in 0cb380f042a2b4f746392b0880a789264365664b